src/main/java/uk/gov/hmcts/darts/dailylist/mapper/DailyListMapper.java

```diff
@@ -22,16 +22,26 @@ public class DailyListMapper {
 
     private final ObjectMapper objectMapper;
 
-
     public DailyListEntity createDailyListEntity(DailyListPostRequest postRequest, CourthouseEntity courthouse) {
-
         DailyListEntity dailyListEntity = new DailyListEntity();
         dailyListEntity.setXmlContent(postRequest.getDailyListXml());
         updateDailyListEntity(postRequest, courthouse, dailyListEntity);
+
         return dailyListEntity;
     }
 
-    public void updateDailyListEntity(DailyListEntity dailyListEntity, DailyListJsonObject dailyList, CourthouseEntity courthouse) {
+    public void updateDailyListEntity(DailyListPostRequest postRequest, CourthouseEntity courthouse, DailyListEntity dailyListEntity) {
```

Comment:
I'm struggling to see how this is required? There is no hard-link between a courthouse entity and the listing courthouse.